### PR TITLE
docs: notice about the installation

### PR DESCRIPTION
**Pull request by TiagoDeveloper**
Information about the need to install the package.

## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by Airblader** [comment]
This PR "needs explanation". It's unclear to me what the added line should tell me(?)

**Comment by TiagoDeveloper** [comment]
You need to install the package as it is a package used in quickstart. you
might want to add this remark in quickstart.
Em 03/06/2019 03:29, "Ingo Bürk" <notifications@github.com> escreveu:

File: packages/elements/PACKAGE.md [hunk 1]
```diff
@@ -5,4 +5,6 @@ A custom element extends HTML by allowing you to define a tag whose content is c
 
 The `createCustomElement()` function provides a bridge from Angular's component interface and change detection functionality to the built-in DOM API. 
 
-For more information, see [Angular Elements Overview](guide/elements).
\ No newline at end of file
+Needs installation.
```
Comment (kapunahelewong) [inline packages/elements/PACKAGE.md]: Consider adding more information here. As to what to add, keep readers in mind who don't know what needs to be installed or how to install it. We want to make sure than even new Angular devs can figure out anything that needs to be done, and as you can see, this is an ongoing endeavor! In terms of grammar, be sure that each sentence is complete with a subject and verb. For example, "Needs installation" lacks a subject -- what needs installation?

**Review by kapunahelewong — CHANGES_REQUESTED** [review]
Hi @TiagoDeveloper, thanks for the PR! Could you add more info in the PR description such as the affected URL (https://angular.io/api/elements) and a description of what your change is about to orient PR reviewers? Also, please clarify your commit message. You could try something like this: "docs: add info about ______ installation to elements API doc" where _____ is the thing to be installed.

**Comment by andrewseguin** [comment]
Closing due to inactivity

**Pull request closed without merging**